Integrations/integration-secdo.yml

```diff
@@ -0,0 +1,135 @@
+description: Secdo's automated incident response platform hunts threats in real time
+  and delivers an endpoint detection and response solution.
+configuration:
+- display: Secdo Server
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Secdo Key
+  name: key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Do not validate server certificate (insecute)
+  name: insecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Company name
+  name: company
+  defaultvalue: ""
+  type: 0
+  required: true
+script:
+  script: |+
+    var sendRequest = function() {
+        var requestUrl = 'https://' + params.server + '/publicapi/v2/load_ioc/';
+        var data = {
+            company: params.company,
+            apikey: params.key,
+            severity: args.severity,
+            ioc_data: args.data,
+            source: 'Demisto',
+            response_plan: args.response,
+            artifacts_type: args.artifact,
+            comment: args.comment
+        };
+
+        var res = http(
+            requestUrl,
+            {
+                Method: 'POST',
+                Headers: {
+                    Accept: ['application/json'],
+                    'Content-Type': ['application/json']
+                },
+                Body: JSON.stringify(data)
+            },
+            params.insecure,
+            params.proxy
+        );
+
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+
+        return res;
+    };
+
+
+    var addIocs = function() {
+        var md = "## Secdo\n";
+        var res =  sendRequest();
+        if (res.StatusCode === 200) {
+            md += 'The IOCs have been successfully added to Secdo.\n';
+        } else {
+            md += 'Opps, it seems that there is some error with adding the IOCs to Secdo.\n'
+        }
+
+        return ( {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': {}, "HumanReadable": md} );
+    };
+
+    switch (command) {
+        case 'test-module':
+            return 'ok';
+        case 'secdo-add-IOCs':
+            return addIocs();
+        default:
+    }
+
+  type: javascript
+  commands:
+  - name: secdo-add-IOCs
+    arguments:
+    - name: artifact
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - mixed
+      - md5hash
+      - filename
+      - fullpath
+      - dest_ip
+      - hostname
+      description: Type of artifacts
+      defaultValue: mixed
+    - name: responce
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - REPORT
+      - SYSLOG
+      - CASE
+      - WHITELIST
+      - SILENT_REPORT
+      description: How Secdo will responde after the IOCs add
```

Comment:
I think the end of the sentence is wrong

Reply:
@dorsha Why? How do you want to change it?

Reply:
First you start the sentence with a question word and there is no question mark.
Second "IOCs add" sounds wrong. Check with Maya

Reply:
Changed